rflx/pyrflx/typevalue.py

```diff
@@ -118,6 +144,14 @@ def assign(self, value: int, check: bool = True) -> None:
             raise ValueError(f"value {value} not in type range {self._first} .. {self._last}")
         self._value = value
 
+    def assign_bitvalue(self, value: str, check: bool = True) -> None:
+
+        for i in value:
```

Comment:
This check should be unnecessary since if you use `int(str, 2)` to parse integers into bit strings. The according error should be clear enough: `ValueError: invalid literal for int() with base 2: '101010sas'`.